syntaxes/src/markdown.glimmer.codeblock.mjs

```diff
@@ -11,6 +11,42 @@ function regexes(extension) {
   };
 }
 
+function codeblock(extension) {
```

Comment:
good job! could is generate command has been executed (to ensure we have same output)?

Reply:
Sorry, I don't quite understand. :smile:

Reply:
Can you run the build script and commit the json files, just to ensure the output is the same :)
The josn files are used directly from this repo in a bunch of different places so we need them committed.

The workflow will do this but it's nice to be able to check the output in the PR too 👍

Reply:
@lifeart just thought, if the output _is_ the same, it won't be able to be committed right? There will be no git diff and nothing to commit?

Reply:
@evoactivity yep, right, just waiting for formal confirm that it's ok :)

Reply:
I've been running the build script during development, but the output is the same, so nothing to commit. :)

Reply:
Sounds like you've found your next contribution 😄